tests/unit_tests/test_first.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def first_process_code():
+    return load_process_code("first")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({'data': []}, None),
+        ({'data': [1,2,3]}, 1),
```

Comment:
Can you also add test for an array will only null values

```
An array without non-null elements resolves always with null.
```

Reply:
Should be fixed